pkg/networkmanager/interfaces.js

```diff
@@ -591,7 +591,7 @@ export function NetworkManagerModel() {
                 peers: get("wireguard", "peers", []).map(peer => ({
                     publicKey: peer['public-key'].v,
```

Comment:
The commit message said "public-key is optional as well, so should this also get the `?` treatment?

Reply:
It said "Missing or invalid public-key is equivalent to no peer", for invalid/missing `public-key` the `.map(...)` function [here](https://github.com/cockpit-project/cockpit/pull/19395/files#diff-e6968ba34dad066826484f30b54426b4891fbd5e72d45297d295a8f46086e3c8R591) wouldn't even run, as it'd be an empty array of peers.

Reply:
Ah, sorry for the misunderstanding.